### DISCUSSION TITLE
Nickelodeon Brasil

### DISCUSSION THREAD

**Discussion by cooldude2144** [iptv-org/community#700] [Q&A]
**Channel Name:** Nickelodeon HD
**Channel ID:** Nickelodeon.br
**Country:** Brazil
**Language:** Spanish, Portugese, English
**Website:** xxx
**Stream URL (optional):** xxx
**Notes (optional):** xxx

**Comment by freearhey** [comment]
Closed in favor of https://github.com/iptv-org/iptv/issues/19922
